app/models/internship_offer.rb

```diff
@@ -55,6 +55,14 @@ class InternshipOffer < ApplicationRecord
     joins(:weeks).where(weeks: {id: weeks.ids}).distinct
   }
 
+  scope :older_than, -> (week:) {
+    joins(:weeks).where("weeks.year > ? OR (weeks.year = ? AND weeks.number > ?)", week.year, week.year, week.number)
```

Comment:
("weeks.year > ? OR (weeks.year = :year AND weeks.number > :number)", {year: week.year, number: week.number })